quiz.rb

```diff
@@ -14,24 +14,56 @@
 # Take a string and return an array of normalized words.
 # normalized words contain no punctuation and are all lower case
 def normalize_words words
+  return words.downcase.gsub(/[^a-z ]/, '').split(' ')
 end
 
 # Question 2
 # Take a string and return the count of normalized words
 def word_count words
+  words.downcase.gsub(/[^a-z ]/, '').split(' ')
+  return words.length
 end
 
 # Question 3
 # Take a string and return and array of the unique normalized words.
 def unique_words words
+  wordHash = {}
+  wordArr = words.downcase.gsub(/[^a-z ]/, '').split(' ')
+  i = 0;
+  while (i < wordArr.length)
+    i += 1
+  wordHash[wordArr[i]] = true
+  end
+  return wordHash
```

Comment:
good job replicating the javascript way to do this problem. if you wanted to return an array, you could use `.keys`